integration/integrationAccuracyExperiment.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Thoughts about integration (for error evalutaion of numerical solutions).
+
+Created on Mon Dec 19 13:21:36 2022
+
+@author: maxvondanwitz
+"""
+import numpy as np
+from sympy import symbols, integrate, sqrt
+from sympy.plotting import plot3d
+from sympy import lambdify
+
+
+def monteCarloIntegration(nn, runs):
+    from numpy import linalg as la
+    import random
+
+    res = np.zeros(runs)
```

Comment:
I think this is unnecessary. You can just say

res = la.norm(np.array(vals, dtype=float)) / nn